note/src/Note.ts

```diff
@@ -0,0 +1,24 @@
+import './Note.styles.css';
+
+const colors = {
+  info: '#54AEFF66',
+  warning: '#ECE81A66',
+  error: '#FF818266',
+};
+
+/**
+ * Component used to call out information in documentation pages.
+ */
+export class Note extends HTMLElement {
+  connectedCallback() {
+    const variant = this.getAttribute('variant');
+    const color = this.getAttribute('color') || colors[variant] || colors.info;
+    this.style.setProperty('--dockit-note-color', color);
+
+    this.innerHTML = /*html*/ `
+      <div class="container">
```

Comment:
if this was ShadowDOM, you could use `::part` for theming too like in the case of dockit-do/dont
good for consistency also and I think there is not reason not to use Shadow DOM here
IMO it should be the default and only in specific cases where Light DOM is absolutely necessary, we need to use that instead
ShadowDOM gives styles scoping which is extremely important for dockit components where chance of style clash with other DS is high